docs/getting-started-logging.md

```diff
@@ -13,15 +13,15 @@ We will create the `loki-stack` component in the following diagram.
 | +------------------------------------+                                                           |
 | |             loki-stack             |                                                           |
 | |                                    |                                       +-----------------+ |
-| | +--------------+  +--------------+ | ---------------------(Logging)------> | Scalar Products | |
+| | +--------------+  +--------------+ | ------------------(Log)-------------> | Scalar Products | |
```

Comment:
```suggestion
| | +--------------+  +--------------+ | <-----------------(Log)-------------- | Scalar Products | |
```
It seems that the direction of the arrow of `Log` is different between this docs and the following docs.

https://github.com/scalar-labs/helm-charts/blob/ad6f94a6244462110f72a47a3b89fc4a089a9dfd/docs/getting-started-scalar-manager.md

If this arrow means `collecting logs`, I think the left arrow is correct.

Reply:
Sorry, I forgot to change the arrow direction here.